### ISSUE TITLE
Test Code for Sisimai::DateTime Fails

### ISSUE THREAD

**Issue by azumakuniyuki** [sisimai/p5-Sisimai#297]
```
#   Failed test at t/013-datetime.t line 112.
#          got: '1541289600'
#     expected: '1541376000'
 ***warning: Strange date format [Thu, 13 Cat 2000 22:22:22 +2222] at lib/Sisimai/DateTime.pm line 441.
 ***warning: Strange date format [Thu, 22 Jun 2001 32:40:29 +0900] at lib/Sisimai/DateTime.pm line 441.
 ***warning: Strange date format [Thu, 25 Jul 1995 00:86:00 +0900] at lib/Sisimai/DateTime.pm line 441.
 ***warning: Strange date format [Thu, 31 Aug 2013 11:22:73 +0900] at lib/Sisimai/DateTime.pm line 441.
 ***warning: Strange date format [Thu, 36 Sep 2009 11:22:33 +0900] at lib/Sisimai/DateTime.pm line 441.
# Looks like you failed 1 test of 318.
t/013-datetime.t .................... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/318 subtests 
t/014-time.t ........................ ok
```
http://www.cpantesters.org/cpan/report/0f9cf06a-df25-11e8-a7e2-b0ce62b887d7

**Comment by azumakuniyuki** [comment]
24 hours differ
```
% echo '1541289600 - 1541376000' |bc
-86400
```

**Comment by azumakuniyuki** [comment]
line 112 of t/013-datetime.t is for Sisimai::DateTime->o2d() but the method is not used in Sisimai.
```
% find t xt lib -type f -exec grep 'o2d' {} /dev/null \;
t/013-datetime.t:        'o2d', 'parse', 'abbr2tz', 'tz2second', 'second2tz',
t/013-datetime.t:            $date = $v->o2d($e);
t/013-datetime.t:            $date = $v->o2d($e);
lib/Sisimai/DateTime.pm:sub o2d {
lib/Sisimai/DateTime.pm:    #   o2d(1)      #=> 2015-11-03
lib/Sisimai/DateTime.pm:    #   o2d(2,'/')  #=> 2015/11/02
```

**Comment by azumakuniyuki** [comment]
Sisimai::DateTime->o2d will be removed in the next release.

**Comment by azumakuniyuki** [comment]
May be fixed on v4.24.1
